addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/WebSocketMessage.java

```diff
@@ -433,4 +433,8 @@ public Mode getProxyMode() {
     public String toString() {
         return "WebSocketMessage#" + getMessageId();
     }
+
+    public static int[] getOpcodes() {
```

Comment:
Does not address the problem, the array can still be modified. Since the APIs are already being broken better use an unmodifiable list.